.github/ISSUE_TEMPLATE/general-support.yml

```diff
@@ -1,11 +1,11 @@
 name: General support
 description: Suggestions that are not related to bugs or new features
-title: 'auro-formvalidation: [issue]'
+title: 'wc-generator: [issue]'
 labels:
   - 'Type: Question'
   - 'Type: Feature'
   - not-reviewed
-  - 'auro-formvalidation'
+  - wc-generator
```

Comment:
revert

Reply:
Why would this repo title the tickets as though they were part of the wc-generator? This is not a pattern we follow anywhere else other than the actual generator.